rust/src/applayerpgsql/logger.rs

```diff
@@ -0,0 +1,36 @@
+/* Copyright (C) 2018 Open Information Security Foundation
```

Comment:
Should the directory be named simply pqsql ?

Reply:
There should also be update in doc/userguide

Reply:
Is there no detection yet ? just logging ?

Reply:
That's right, no detection yet, just logging, a simplified version, for now. 
Indeed, I think the directory can be changed, I'll do that. And I'll also add the doc/userguide to my todo list!